src/web/components/Card/components/LI.tsx

```diff
@@ -56,6 +61,7 @@ type Props = {
 	padSides?: boolean; // If this LI directly wraps a card this should be true
 	bottomMargin?: boolean; // True when wrapping a card in a column and not the last item
 	showTopMarginWhenStacked?: boolean;
+	snapAlignStart?: boolean; // True when snapping card when scrolling e.g. in carousel
```

Comment:
+1 for the comment here and +2 for keeping it generic so it works for other use cases